components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -1418,6 +1418,13 @@
             " SELECT UUID, REG_NAME, REG_TYPE, MODE, TENANT_ID, REG_OWNER, READ_ROLE, WRITE_ROLE FROM " +
                     "ENDPOINT_REG WHERE TENANT_ID = ?";
 
+    public static final String GET_ALL_ENTRIES_OF_ENDPOINT_REGISTRY =
+            "SELECT E.UUID, E.ENTRY_NAME, E.DEFINITION_TYPE, E.DEFINITION_URL, E.METADATA, " +
```

Comment:
since UUID is unique do we need to associate the Registry table here? Just fetching from the Entry table would be sufficient right?

Reply:
foreign key is created using the auto generated id not using uuid

Reply:
ignore the above comment.